src/config.js

```diff
@@ -1,3 +1,7 @@
+import fs from 'fs';
+import * as dotenv from 'dotenv';
+import * as dotenvExpand from 'dotenv-expand';
```

Comment:
we you really need this? I dont see dotenv-expand featured used in any of the committed files. it add complexity to this project and enabled more complex patterns that I don't think we should have in env vars. KISS.